<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="WKWIN6024718" name="Regression Test for Class 1" tests="2" failures="2" timestamp="21 May 2020 05:29:21 GMT" time="86.408" errors="0">
  <testcase name="verifyLogin" time="37.271" classname="publicisSapient.testScripts.HomePageTest">
    <failure type="java.lang.AssertionError" message="expected [fail] but found [pass]">
      <![CDATA[java.lang.AssertionError: expected [fail] but found [pass]
	at publicisSapient.testScripts.HomePageTest.verifyLogin(HomePageTest.java:38)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- verifyLogin -->
  <testcase name="verifyLogin" time="42.622" classname="publicisSapient.testScripts.HomePageTest">
    <failure type="java.lang.AssertionError" message="expected [fail] but found [pass]">
      <![CDATA[java.lang.AssertionError: expected [fail] but found [pass]
	at publicisSapient.testScripts.HomePageTest.verifyLogin(HomePageTest.java:38)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- verifyLogin -->
</testsuite> <!-- Regression Test for Class 1 -->
